<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import MealMaster Recipes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="anymeal.qrc">
    <normaloff>:/images/anymeal.png</normaloff>:/images/anymeal.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="encoding_label">
       <property name="text">
        <string>En&amp;coding</string>
       </property>
       <property name="buddy">
        <cstring>encoding_combo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="encoding_combo">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>CP850</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO-8859-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO-8859-15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>US-ASCII</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="error_file_label">
       <property name="text">
        <string>&amp;Error File</string>
       </property>
       <property name="buddy">
        <cstring>error_file_edit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="error_file_edit">
         <property name="text">
          <string>/tmp/errors.mmf</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_button">
         <property name="text">
          <string>&amp;Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="anymeal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
